samples/apps/copilot-chat-app/webapp/src/libs/connectors/useConnectors.ts

```diff
@@ -1,55 +1,10 @@
-import { useMsal } from '@azure/msal-react';
-import { Constants } from '../../Constants';
 import { useAppSelector } from '../../redux/app/hooks';
```

Comment:
I say move `getEnabledPlugins` to useChat (the only place it's called) and get rid of useConnectors.ts altogether

I created this file before we had a clear story with Plugins and Planner so there's no need for it anymore since we're not invoking any Plugins directly

Also the word Connectors in general needs to be scrubbed from the front end